utils.py

```diff
@@ -328,3 +329,62 @@ def generate_data_frame_chunks(meta, chunk_size):
 
 def categorize_image(image, channel_axis=0):
     return np.argmax(image, axis=channel_axis)
+
+
+def coco_evaluation(gt_filepath, prediction_filepath, image_ids, category_ids):
+
+    coco = COCO(gt_filepath)
+    coco_results = coco.loadRes(prediction_filepath)
+    cocoEval = COCOeval(coco, coco_results)
+    cocoEval.params.imgIds = image_ids
+    cocoEval.params.catIds = category_ids
+    cocoEval.evaluate()
+    cocoEval.accumulate()
+    cocoEval.summarize()
+
+    return cocoEval.stats[1], cocoEval.stats[8]
+
+
+def generate_prediction(meta_data, pipeline, logger, category_ids, chunk_size):
```

Comment:
@taraspiotr I think both `generate_prediction` and `_generate_prediction` should be in `main.py` because otherwise we should move a lot of other things away from `main.py` . What do you think?

Reply:
@jakubczakon True, I have a tendency to do that. I moved them to `main.py`.